### PR TITLE
docs: update simulated annealing

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

File: docs/source/workflows.rst [hunk 1]
```diff
@@ -813,6 +813,13 @@ and it finally performs a minimization for the final fine tuning of the optimum
         "number_of_molecules": 1
     }
 
+
+  In particular:
+
+  * ``host/ads`` describes the interaction between host and (all) adsorbates, ``ads/ads`` the interaction between adsorbates
+  * The binding energy is the final value of ``energy_host/ads_coulomb_final`` (you still need to divide by ``number_of_molecules``)
+  * ``energy_ads/ads_coulomb_final`` and ``energy_ads/ads_tot_final`` may be non-zero even for a single molecule due to rounding errors in Ewald summation
```
Comment (mpougin) [inline docs/source/workflows.rst:821]: and ``energy_ads/ads_vdw_final``

**Review by mpougin — DISMISSED** [review]
It's much clearer now. Is there a specific reason, why the last temperature point in this 50K and not 0K?

**Comment by ltalirz** [comment]
> It's much clearer now. Is there a specific reason, why the last temperature point in this 50K and not 0K?

I'm not an MC person, but I guess a 0K Monte-Carlo calculation is essentially a very inefficient geometry optimization (you cannot accept any moves that increase the energy)

File: docs/source/workflows.rst (hunk 1 above)
Comment (ltalirz) [reply docs/source/workflows.rst:821]: well, the van-der-waals energies are usually computed with cutoffs, thus they should be exactly zero for a single molecule (which they are in your example as well)

Comment (danieleongari) [reply docs/source/workflows.rst:821]: > (you still need to divide by ``number_of_molecules``) 

It depends what you want to get, I wouldn't write it as a mandatory sentence
I would write something as:

> In the case of ``number_of_molecules``>1, you may want to obtain an average binding energy per molecule by dividing  this value by the number of molecules.

**Review by danieleongari — APPROVED** [review]

**Comment by danieleongari** [comment]
Ah, ok, you merged in the meanwhile I was reviewing: that's ok my comment was minor

**Comment by ltalirz** [comment]
> Ah, ok, you merged in the meanwhile I was reviewing: that's ok my comment was minor

Sorry, I'll add it in the next update of the docs

**Pull request merged**